Atomic/syscontainers.py

```diff
@@ -57,6 +58,33 @@ def __init__(self):
     def get_atomic_config_item(self, config_item):
         return util.get_atomic_config_item(config_item, atomic_config=self.atomic_config)
 
+    # Try different strategies to do a syncfs:
+    # 1) Load it from libc.
+    # 2) Run "sync --file-system", supported by coreutils sync.
+    # 3) Use os.sync, available from Python 3.3.
+    # 4) Run "sync".
```

Comment:
Holy sync batman!  Do we really need any strategy other than the libc one?  Why wouldn't ctypes work everywhere?

BTW I think we really want `syncfs` and not `sync` because the latter doesn't actually report errors.